pkg/util/exec_windows.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2021 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"syscall"
+)
+
+// Exec creates a new process with the specified arguments.
+func Exec(name string, arg ...string) *exec.Cmd {
+	// Windows does not handle relative path names in exec very well.
+	name = filepath.Clean(name)
+	cmdArgs := arg
+
+	// Unlike Linux Windows not create a shell context
+	if strings.ToLower(filepath.Ext(name)) == ".cmd" {
```

Comment:
Can you elaborate a bit more here?

Reply:
I've added more information. Also added powershell support because that's generally preferred as it provides more flexibility than cmd.exe.